Modules/IO/LSM/src/itkLSMImageIO.cxx

```diff
@@ -357,7 +357,7 @@ void LSMImageIO::Write(const void *buffer)
 
     if ( compression == COMPRESSION_JPEG )
       {
-      TIFFSetField(tif, TIFFTAG_JPEGQUALITY, 75); // Parameter
+      TIFFSetField(tif, TIFFTAG_JPEGQUALITY, this->GetCompressionLevel()); // Parameter
```

Comment:
This will change default JPEG quality from 75 to 30, which is probably not desired.